<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_Test">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>89</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_SendMap">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>60</y>
      <width>89</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send map</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_Screen1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
